core/templates/dev/head/forms/formBuilder.js

```diff
@@ -627,6 +641,10 @@ oppia.factory('rteHelperService', [
           el.attr(
             $filter('camelCaseToHyphens')(attrName) + '-with-value',
             oppiaHtmlEscaper.objToEscapedJson(customizationArgsDict[attrName]));
+          if (componentDefn.name === 'image' && attrName === 'width') {
```

Comment:
@AllanYangZhou do we actually need to special-case this? Prefer not to put special cases for individual components in general code if possible...